build.gradle

```diff
@@ -250,6 +250,7 @@ dependencies {
         exclude group: 'org.apache.logging.log4j', module: 'log4j-api'
     }
     compile 'org.jooq:jool-java-8:0.9.14'
+    compile 'com.github.hmcts:ccd-case-document-am-client:1.7.0'
```

Comment:
Can you bump to 1.7.1 please as I've released a new version today.

Reply:
Fixed in 70dce10

Reply:
sorry i still can't see